### PR TITLE
feat: add react-intl

### LINKED ISSUE #1410: Add React intl package

### PR DESCRIPTION
**Pull request by kaitmore**
Resolves #1410

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -10,17 +11,36 @@ import AppFooter from "./components/Footer";
 import Notification from "./components/Notification";
 import Main from "./components/Main";
 import store from "./store";
+import { flattenMessages } from "./utils";
+import translations from "./translations";
 import "./services";
+
+import en from "react-intl/locale-data/en";
```
Comment (spmcbride1201) [inline src/index.js]: This is fine for now. Depending on the number of locales, we'll want to dynamically load these in.

File: src/index.js [hunk 2]
```diff
@@ -10,17 +11,36 @@ import AppFooter from "./components/Footer";
...
-        <Notification />
-        <AppHeader />
-        <Main />
-        <AppFooter />
-      </AppContainer>
-    </Router>
-  </Provider>,
+  <IntlProvider
+    locale={locale}
+    messages={flattenMessages(translations[locale])}
```
[the remark this answers is not in the document]
Comment (spmcbride1201) [reply src/index.js]: This looks relevant: https://github.com/yahoo/react-intl/issues/1089

File: src/components/Main/scenes/InstanceView/scenes/GoInstanceView/scenes/Summary.js [hunk 3]
```diff
@@ -42,16 +48,27 @@ function SummaryGrid({ metrics, selectedInstanceID, selectedServiceSlug }) {
     "B",
     "GB"
   );
+  console.log(startTime);
```
Comment (spmcbride1201) [inline src/components/Main/scenes/InstanceView/scenes/GoInstanceView/scenes/Summary.js]: [¬º-°]¬ [¬º-°]¬ [¬º-°]¬      ─╤╦︻  (̿▀̿ ̿Ĺ̯̿̿▀̿ ̿)̄

File: src/translations.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      hostCPUUsage: "Host CPU Utilized",
+      hostCPUUsageDetail:
+        "{count, plural, one {# Core on Host} other {#  Cores on Host}}",
+      memoryUsage: "Memory Utilized",
+      memoryUsageDetail: "{hostMemoryAvail} GB Free on Host",
+      vitals: "Vitals",
+      statistics: "Statistics"
+    }
+  },
+  "en-ES": {
```
[the remark this answers is not in the document]
Comment (kaitmore) [reply src/translations.js]: I understand the format, it was just a typo. I'm going to remove this anyways. 👍

**Pull request merged**